shared/macros/srg_requirement.jinja

```diff
@@ -0,0 +1,10 @@
+{{#
+    How to fix an audit rule that watches a file.
```

Comment:
Something like this would be better: 
Generate a SRG requirement text for audit file watch rules

Reply:
I have addressed the feedback on Matthew's behalf